### ISSUE TITLE
Wrangler ORTE does not work with 16 nodes

### ISSUE THREAD

**Issue by iparask** [radical-cybertools/radical.pilot#1553]
I am trying to run a 16node RP script and it fails. I am using the latest release and the configuration is `wrangler_orte`.

The error I am getting is:`agent_0.idler._check_units_cb: ERROR   : invalid state:`

How should I fix this?

**Comment by andre-merzky** [comment]
Do you have stack infos and a set of logfiles, by any chance?  Just from the above its hard to say anything I'm afraid...

**Comment by iparask** [comment]
Yes I do. Which log file you want?

**Comment by andre-merzky** [comment]
If its ORTE related, then only the agent sandbox will be of relevance I assume.  Otherwise I don't really know before looking through them I'm afraid...

Cheers, Andre

**Comment by andre-merzky** [comment]
Uhm, to repeat: if it's an ORTE problem, then only the *agent* side is relevant...

**Comment by iparask** [comment]
All log files are there.

**Comment by andre-merzky** [comment]
What is the `radical-stack` you used?  Thanks!

**Comment by iparask** [comment]
```
  python               : 2.7.13
  pythonpath           :
  virtualenv           : rp0.47

  radical.analytics    : 0.47.0
  radical.pilot        : 0.47.1
  radical.utils        : 0.47
  saga                 : 0.47
```

**Comment by andre-merzky** [comment]
Those log files don't fit together:

```
 two  root  …/WeakScalingUpTo32Nodes/test2/pilot.0000  [master *] # l agent_0.log ../agent_0.log
-rw------- 1 iparask iparask 922599 Feb  3 22:43 ../agent_0.log
-rw------- 1 iparask iparask  57632 Feb  3 22:41 agent_0.log
```

**Comment by iparask** [comment]
I do not know how that happened. Under `WeakScalingUpTo32Nodes` there is a folder named `Test384cores`. You can find everything there.

**Comment by andre-merzky** [comment]
That session contains the same error we discussed in slack:

```
930 --> Submission error: all jobs must have a queue name specified with "-p"
```

which seems unrelated to the incorrect state error in this ticket.  Can you please open a separate ticket for that issue?

**Comment by andre-merzky** [comment]
Well, the error you see is:

```
invalid state: {u'_id': u'unit.000000',
...
 u'state': 'FAILED',
...
```

and `FAILED` is indeed an invalid unit state for the agent to handle.  This seems caused by the `-p` issue from above, which is in the `UMGR_STAGING_INPUT` component.  So, that is what needs  fixing.

**Comment by iparask** [comment]
That being said the job has started, since it has all the log files. Ticket #1554 is opened for that error

**Comment by iparask** [comment]
This is fixed by #1554 fix. Closing.
